how-to/backups/install-gitolite.md

```diff
@@ -1,88 +1,148 @@
 (install-gitolite)=
 # How to install and configure gitolite
 
-{term}`Gitolite` provides a traditional source control management server for git, with multiple users and access rights management. 
+{term}`Gitolite` provides a traditional source control management server for git, with multiple users and access rights management.
+
+It provides access so you can add remote repos in the form of `git@yourserver:some/repo/path`.
+
 
 ## Install a gitolite server
 
 Gitolite can be installed with the following command:
 
 ```bash
+# it will ask for a path or content of your admin ssh key - instead
+# we recommend leaving it empty and to do the manual setup below.
 sudo apt install gitolite3
 ```
 
+The official documentation is at https://gitolite.com/gitolite/
+
 ## Configure gitolite
 
-Configuration of the gitolite server is a little different that most other servers on Unix-like systems, in that gitolite stores its configuration in a git repository rather than in files in `/etc/`. The first step to configuring a new installation is to allow access to the configuration repository.
+Gitolite stores its configuration in a git repository (called `gitolite-admin`), so there's no configuration in `/etc`.
+This configuration repository manages all other git repos, users and their permissions.
 
-First of all, let's create a user for gitolite to use for the service:
+Create a `git` user for gitolite to use for the service:
 
 ```bash
-sudo adduser --system --shell /bin/bash --group --disabled-password --home /home/git git
+# you can adjust the git repo storage path as the home directory
+sudo useradd adduser --system --home /home/git git
```

Comment:
important - question:

useradd + adduser - that seems wrong or am I missing the point?